README.md

```diff
@@ -1,185 +1,212 @@
 
-#### 3.1.2 Jira
+// TODO: Add 1 or 2 screenshots
 
-As a Jira administrator, you have two steps to configure the plugin:
+The supported Jira fields are:
 
-#### 3.1.3 Jira - Deploy via Jira AppConnect
+  - **Project Picker**: Custom fields and the built-in **Project** field.
+  - **Single-Line Text**: Custom fields, and built-in fields such as **Summary**, **Environment**.
+  - **Multi-Line Text**: Custom fields, and built-in fields such as **Description**.
+  - **Single-Choice Issue**: Custom fields, and built-in fields such as **Issue Type** and **Priority**. 
 
-1. As a Jira System Administrator, go to **Jira Settings > Apps > Manage Apps**. 
-  - For older versions of Jira, go to **Administration > Applications > Add-ons > Manage add-ons**
+#### 1.2.2 Attach Messages to Jira Issues
 
-2. Click **Settings** at bottom of page and enable development mode, and apply this change.
-  - Enabling development mode allows you to install apps that are not from the Atlassian Marketplace.
-  - Mattermost has opted not to publish to Atlassian Marketplace, as we don’t have an efficient way to provide a callback URL for the app in the marketplace, to enable user-specific interactions between the Mattermost server and Jira cloud instance.
+Keep all information in one place by attaching parts of Mattermost conversations in Jira issues as comments. To attach a message, click the **More Actions** (...) option of any message in the channel (available when you hover over a message), then select **Attach to Jira Issue**.
+
+// TODO: Add screenshot
 
-3. Click **Upload app**, then enter the Atlassian Connect app descriptor in the form https://SITEURL/plugins/jira//ac/atlassian-connect.json where `SITEURL` is your [Mattermost Site URL](https://docs.mattermost.com/administration/config-settings.html#site-url). Select **Upload**.
+Then, on the resulting issue creation dialog, select the project and the issue you want to attach it to. You may search for issues containing specific text.
 
-4. Wait for the app to install.
+// TODO: Add screenshot
 
-You're all set. Users can now connect their Mattermost account with Jira using `/jira connect`.
+Click **Attach** and the message is now attached to the Jira issue.
 
-#### 3.1.4 Jira - Configure Webhooks
+// TODO: Add screenshot from Jira UI
 
-Only the Jira System Admin has permissions to link a Jira project to a Mattermost channel via webhooks. We are planning to allow anyone with access to subscribe a Jira project to a Mattermost channel via `/jira subscribe`.
+#### 1.2.3 Transition Jira issues
+
+Transition issues without the need to switch to your Jira project. To transition an issue, use the `/jira transition <issue-key> <state>` command.
+
+For instance, `/jira transition MM-1234 done` transitions the issue key **MM-1234** to **Done**.
+
+Note that states and issue transitions are based on your Jira project workflow configuration.
+
+## 2. Installation
```

Comment:
I will consider condensing the `Installation` guide to 

1. Shorten the steps to configure the webhook in Jira
2. Ask admins to just follow the steps posted after `/jira install` rather than writing all the steps in this doc.